tikv/tikv
# copr: Redact remaining Coprocessor logs

Signed-off-by: Breezewish <breezewish@pingcap.com>



### What problem does this PR solve?

Issue Number: close #xxx 

Problem Summary: This PR provide log redact for remaining Coprocessor logs.

### What is changed and how it works?

What's Changed: Use log_wrapper to provide log redact.

### Related changes

- Need to cherry-pick to the release branch

### Check List 

Tests 

- Manual test

I confirm that the remaining Coprocessor log is already redacted:

```
[2020/12/14 13:30:10.778 +08:00] [WARN] [endpoint.rs:639] [error-response] [err="Key is locked (will clean up) primary_lock: ? lock_version: 421507579421655041 key: ? lock_ttl: 3000 txn_size: 1 min_commit_ts: 421507579421655042"]
[2020/12/14 13:30:10.970 +08:00] [INFO] [tracker.rs:221] [slow-query] [perf_stats.internal_delete_skipped_count=39] [perf_stats.internal_key_skipped_count=98] [perf_stats.block_read_byte=0] [perf_stats.block_read_count=0] [perf_stats.block_cache_hit_count=0] [scan.range.first="Some(start: ? end: ?)"] [scan.ranges=1] [scan.total=31] [scan.processed=5] [scan.is_desc=false] [tag=select] [table_id=49] [txn_start_ts=421507579447869444] [total_process_time=1ms] [handler_build_time=0ns] [wait_time.snapshot=0ns] [wait_time.schedule=0ns] [wait_time=0ns] [total_lifetime=1ms] [remote_host=ipv4:127.0.0.1:55582] [region_id=2]
```

### Release note 

- No release note.
